content/blog/pulumi-2-0-roadmap/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Pulumi 2.0 Roadmap"
```

Comment:
What I don't quite understand after reading this blog: why do those changed require 2.0? Are they breaking 1.0 and where?